### PR TITLE
non-acter spaces support 

### PR DESCRIPTION
**Pull request by gnunicorn**
- [x] show indicator in the top-bar of the space
- [x] on non-acter spaces:
  - [x] show warning above the "About" in the overview
      - [x] if the permissions allow for it, offer to convert to an acter space  
  - [x] disappear top-bar features that are acter-only 
  - [x] hide acter-only cards in overview

### CODE REVIEW

File: app/lib/features/space/providers/space_navbar_provider.dart [hunk 1]
```diff
@@ -6,88 +7,93 @@ import 'package:flutter/material.dart';
...
 import 'package:flutter_svg/flutter_svg.dart';
 
+typedef MakeIconFn = Widget Function(BuildContext);
+
 class TabEntry {
   final Key key;
   final String label;
   final String target;
-  final Widget icon;
+  final MakeIconFn makeIcon;
```
Comment (gtalha07) [inline app/lib/features/space/providers/space_navbar_provider.dart:16]: Why just not have it with ```Icon```?

File: app/lib/features/space/widgets/non_acter_space_card.dart [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import 'package:go_router/go_router.dart';
+
+class NonActerSpaceCard extends ConsumerWidget {
+  final String spaceId;
+  const NonActerSpaceCard({super.key, required this.spaceId});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final myMembership = ref.watch(spaceMembershipProvider(spaceId));
+    const fallback = Text(
```
Comment (gtalha07) [inline app/lib/features/space/widgets/non_acter_space_card.dart:14]: I feel like all these plain text strings should be resided in separate class under ```constants``` directory. But right now we can left it as it is till we properly do localisation.

File: app/lib/features/space/widgets/non_acter_space_card.dart [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            ),
+            const SizedBox(height: 10),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void upgradeSpace(BuildContext context, WidgetRef ref) async {
+    customMsgSnackbar(context, 'Converting to acetr space');
```
Comment (gtalha07) [inline app/lib/features/space/widgets/non_acter_space_card.dart:54]: Minor typo :)

Comment (gnunicorn) [reply app/lib/features/space/widgets/non_acter_space_card.dart:54]:
```suggestion
    customMsgSnackbar(context, 'Converting to acter space');
```

File: app/lib/features/space/widgets/non_acter_space_card.dart (hunk 2 above)
Comment (gnunicorn) [reply app/lib/features/space/widgets/non_acter_space_card.dart:14]: +1 for translations ... but until then ...

**Pull request merged**